addons/l10n_co/data/template/account.tax-co.csv

```diff
@@ -1,225 +1,225 @@
+"","","","","","","","","","tax","refund","","","",""
+"l10n_co_tax_12","1","2.85% Rte","-2.85","percent","purchase","tax_group_r_iva_285","","","base","invoice","","RteIVA 15% over 19% VAT (-2.85%)","-2.85%","RteIVA 15% sobre el 19% IVA"
+"","","","","","","","","","tax","invoice","co_puc_236700","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_236700","","",""
+"l10n_co_tax_13","1","0.75% Rte","-0.75","percent","purchase","tax_group_r_iva_075","","","base","invoice","","RteIVA 15% over 5% VAT (-0.75%)","-0.75%","RteIVA 15% sobre el 5% IVA"
+"","","","","","","","","","tax","invoice","co_puc_236700","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_236700","","",""
+"l10n_co_tax_14","1","0.75% Rte C","0.75","percent","purchase","tax_group_r_iva_075","False","","base","invoice","","RteIVA 15% over 5% VAT Counterpart (0.75%)","0.75%","RteIVA 15% sobre el 5% IVA Contrapartida"
+"","","","","","","","","","tax","invoice","co_puc_240821","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_240821","","",""
+"l10n_co_tax_15","1","2.85% Rte C","2.85","percent","purchase","tax_group_r_iva_285","False","","base","invoice","","RteIVA 15% over 19% VAT Counterpart (2.85%)","2.85%","RteIVA 15% sobre el 19% IVA Contrapartida"
+"","","","","","","","","","tax","invoice","co_puc_240821","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_240821","","",""
+"l10n_co_tax_16","1","0.1% F C","-0.1","percent","purchase","tax_group_r_ren_01","","","base","invoice","","RteFte Fuel Purchase (-0.1%)","-0.1%","RteFte Compra Combustibles 0.1%"
+"","","","","","","","","","tax","invoice","co_puc_236540","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_236540","","",""
+"l10n_co_tax_17","1","0.5% Rte C","-0.5","percent","purchase","tax_group_r_ren_05","","","base","invoice","","RteFte Buy Coffee (-0.5%).","-0.5%","RteFte Compra Cafe 0.5%"
+"","","","","","","","","","tax","invoice","co_puc_236540","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_236540","","",""
+"l10n_co_tax_18","1","10% Rte C N P","-10.0","percent","purchase","tax_group_r_ren_10","","","base","invoice","","RteFte Commissions Natural Person (-10%)","-10%","RteFte Comisiones Persona Natural 10%"
+"","","","","","","","","","tax","invoice","co_puc_236520","","",""
+"","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","tax","refund","co_puc_236520","","",""
+"l10n_co_tax_19","1","2.5% Rte D","-2.5","percent","purchase","tax_group_r_ren_25","","","base","invoice","","RteFte Purchases Declarants (-2.5%)","-2.5%","RteFte Compras Declarantes 2.5%"
```

Comment:
Consistency: parentheses in the source term, but not in the translation

EDIT: I'll stop copy-pasting this comment, but it happens on multiple subsequent line as well ^^